src/views/RandomView.vue

```diff
@@ -77,6 +80,7 @@
         type: "excuse",
         isFavorited: false
       }
+      notificationSound.play();
```

Comment:
...I am going to start doing so much stupid audio shit now that I know how to do it. Thank you Cyanne!